docs/administering-lagoon/installing-lagoon-on-openshift.md

```diff
@@ -4,7 +4,7 @@ Lagoon is not only capable of deploying into OpenShift, it actually runs in Open
 
 Luckily we can use the local development environment to kickstart another Lagoon in any OpenShift, running somewhere in the world.
 
-Check the [Requirements for OpenShift by Lagoon](https://github.com/AlannaBurke/lagoon/tree/9331f688073fb0d34541bf3fb5fe65d2a2c665aa/docs/administering-lagoon/openshift_requirements.md) before continuing.
```

Comment:
you have NO IDEA how many times i searched and fixed these links. how did this get here???